share/revng/test/tests/daemon/test.py

```diff
@@ -129,9 +129,9 @@ def error_handler(e: BaseException):
 
     binary = pytestconfig.getoption("binary")
     tracing = TraceConfig()
-    tracing.on_request_headers_sent.append(header_trace)
-    tracing.on_request_chunk_sent.append(payload_trace)
-    tracing.on_request_end.append(response_trace)
+    tracing.on_request_headers_sent.append(header_trace)  # type: ignore
+    tracing.on_request_chunk_sent.append(payload_trace)  # type: ignore
+    tracing.on_request_end.append(response_trace)  # type: ignore
```

Comment:
This commit is no longer required, this was fixed by the `feature/pypeline` branch